### PR TITLE
DOCS-1901: Fix python error, add cloudmetadata

### PR DESCRIPTION
**Pull request by npentrel**
I think the links will start working tomorrow, so won't merge till then

### CODE REVIEW

File: docs/build/program/apis/robot.md [hunk 1]
```diff
@@ -885,4 +885,28 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- None.
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:901]: @maximpertsov I don't feel like this is correct but there's no info provided in the python sdk docs. I'm a bit confused.

Comment (maximpertsov) [reply docs/build/program/apis/robot.md:901]: yeah it's wrong for sure, I'll make a follow-up to fix

Comment (maximpertsov) [reply docs/build/program/apis/robot.md:901]: https://github.com/viamrobotics/viam-python-sdk/pull/549

File: docs/build/program/apis/robot.md [hunk 2]
```diff
@@ -885,4 +885,63 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:901]:
```suggestion
- [viam.proto.robot.GetCloudMetadataResponse](https://python.viam.dev/autoapi/viam/proto/robot/index.html#viam.proto.robot.GetCloudMetadataResponse): App-related metadata.
```

File: docs/build/program/apis/robot.md [hunk 3]
```diff
@@ -885,4 +885,63 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [viam.proto.robot.GetCloudMetadataResponse](https://python.viam.dev/autoapi/viam/proto/robot/index.html#viam.proto.robot.GetCloudMetadataResponse): App-related metadata.
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:901]:
```suggestion
- [viam.proto.robot.GetCloudMetadataResponse](https://python.viam.dev/autoapi/viam/proto/robot/index.html#viam.proto.robot.GetCloudMetadataResponse): App-related metadata containing the primary org id, location id, and robot part id for a robot running on the Viam app.
```

File: docs/build/program/apis/robot.md [hunk 4]
```diff
@@ -885,4 +885,63 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- None.
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:918]:
```suggestion
- [cloud.Metadata](https://pkg.go.dev/go.viam.com/rdk/cloud): App-related metadata containing the primary org id, location id, and robot part id for a robot running on the Viam app.
```

File: docs/build/program/apis/robot.md [hunk 5]
```diff
@@ -885,4 +885,63 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+
+**Returns:**
+
+- [viam.proto.robot.GetCloudMetadataResponse](https://python.viam.dev/autoapi/viam/proto/robot/index.html#viam.proto.robot.GetCloudMetadataResponse): App-related metadata.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.get_cloud_metadata).
+
+```python
+metadata = robot.get_cloud_metadata()
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:906]:
```suggestion
metadata = robot.get_cloud_metadata()
fmt.Println(metadata.RobotPartID)
fmt.Println(metadata.PrimaryOrgID)
fmt.Println(metadata.LocationID)
```

File: docs/build/program/apis/robot.md [hunk 6]
```diff
@@ -885,4 +885,63 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+- None.
+
+**Returns:**
+
+- None.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/robot#Robot).
+
+```go
+metadata := robot.GetCloudMetadata()
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:923]:
```suggestion
metadata := robot.GetCloudMetadata()
fmt.Println(metadata.RobotPartID)
fmt.Println(metadata.PrimaryOrgID)
fmt.Println(metadata.LocationID)
```

File: docs/build/program/apis/robot.md [hunk 7]
```diff
@@ -885,4 +885,69 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- None.
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:941]:
```suggestion
- [CloudMetadata](https://ts.viam.dev/classes/robotApi.CloudMetadata.html): App-related metadata containing the primary org id, location id, and robot part id for a robot running on the Viam app.
```

File: docs/build/program/apis/robot.md [hunk 8]
```diff
@@ -885,4 +885,69 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+
+For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/RobotClient.html).
+
+```typescript
+// Get the metadata of the machine.
+const metadata = await robot.getCloudMetadata();
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:947]:
```suggestion
const metadata = await robot.getCloudMetadata();
console.log(metadata);
```

**Review by andf-viam — APPROVED** [review]
LGTM!

**Review by maximpertsov — APPROVED** [review]

File: docs/build/program/apis/robot.md [hunk 9]
```diff
@@ -885,4 +885,70 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
...
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [cloud.Metadata](https://pkg.go.dev/go.viam.com/rdk/cloud): App-related metadata containing the primary org id, location id, and robot part id for a robot running on the Viam app.
```
Comment (npentrel) [inline docs/build/program/apis/robot.md:921]:
```suggestion
- [cloud.Metadata](https://pkg.go.dev/go.viam.com/rdk/internal/cloud#Metadata): App-related metadata containing the primary org id, location id, and robot part id for a robot running on the Viam app.
```

**Pull request merged**